qkeras/quantizers.py

```diff
@@ -441,6 +440,18 @@ def min(self):
     else:
       return -1.0
 
+  def range(self):
+    """ Get the range of values that quantized_bits can represent """
```

Comment:
the range of values? or Get the sorted values?

Reply:
it currently outputs a list of all values it can output, unsorted. Should I sort them?

Reply:
Yes. I think it is better to sort them for now. If we want to get the range, we could simply pick the first and last values.
Also change the docstring in this function, it is not "Get the range", it is "Gets the sorted values".

Reply:
The thing is we the list of all possible values that can be outputted by the quantizer in order to create the in_table